roles/openstack-setup/tasks/computes.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: create host aggs
+  os_nova_host_agg:
+    name: "{{ hostvars[item]['compute_ag'] }}"
+    az: "{{ hostvars[item]['compute_ag'] }}"
+    state: present 
+    auth:
+      auth_url: "{{ endpoints.auth_uri }}"
+      project_name: admin
+      username: admin
+      password: "{{ secrets.admin_password }}"
+  run_once: True
+  when: hostvars[item]['compute_ag'] is defined
+  with_items: "{{ groups['compute'] }}"
+
+- name: assign hosts to aggs
```

Comment:
In this PR, it doesn't. We could have a pre-upgrade playbook that checks this thing and has the op adjust the env config to match what's in the cluster, or talk to the customer about changing the AG/AZ set.

And this does nothing if those values aren't defined in the env config, which could be the short-cut for upgrading existing customers.